crates/forge/src/warn.rs

```diff
@@ -186,9 +187,9 @@ mod tests {
 
         assert_stdout_contains(
             stdout,
-            indoc!(
+            formatdoc!(
                 r"
-                    [WARNING] The RPC node with url = http://127.0.0.1:3030/rpc has unsupported version = (0.5.0), use node supporting RPC version 0.6.0
+                    [WARNING] The RPC node with url = http://127.0.0.1:3030/rpc has unsupported version = (0.5.0), use node supporting RPC version {EXPECTED_RPC_VERSION}
```

Comment:
Depending on where the discussion on company's slack regarding our nodes' urls will go, I'd actually suggest this test was not mocked. Since you already use real nodes for some of the tests, it would be trivial for forge/chetnet to test the functionality based on real-life output (especially so that the nodes support more than one spec version). This is something that is not very easy in scnast (devnet only supports one fixed version), so it may be worth to add this here.

Reply:
Changed